src/main/java/teammates/ui/controller/InstructorFeedbackPreviewAsStudentAction.java

```diff
@@ -30,7 +30,7 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                     "Student Email " + previewStudentEmail + " does not exist in " + courseId + ".");
         }
 
-        FeedbackSubmissionEditPageData data = new FeedbackSubmissionEditPageData(account, student, sessionToken);
+        FeedbackSubmissionEditPageData data = new FeedbackSubmissionEditPageData(account, previewStudent, sessionToken);
```

Comment:
Not sure if this is something needs fixing; what is the effect of this?

Reply:
The `student` passed in here is null, so it results in a NPE when previewing the form as a student. Passing in previewStudent fixes this.